lib/linguist/languages.yml

```diff
@@ -4461,6 +4461,24 @@ SRecode Template:
   extensions:
   - ".srt"
   language_id: 335
+SSH Config:
+  type: data
+  group: INI
+  filenames:
+  - "ssh-config"
+  - "ssh_config"
+  - "sshconfig"
+  - "sshd-config"
+  - "sshd_config"
```

Comment:
I counted only 3 such files, but again, [lots of `sshd_config.vim` files](https://github.com/search?p=99&q=filename%3Asshdconfig.vim&type=Code).

Reply:
Same as above, this is a file with the vim configuration syntax, not sshconfig and it's highlighted correctly.

Reply:
Yep, my bad.